src/js/components/CheckBox/stories/Indeterminate.js

```diff
@@ -1,61 +1,52 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import { storiesOf } from '@storybook/react';
 
 import { Box, Grommet, CheckBox } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-class IndeterminateCheckBox extends Component {
-  state = {
-    checked: [],
-    checkboxes: ['fruits', 'vegetables', 'olive oil'],
-  };
+const IndeterminateCheckBox = () => {
+  const [checked, setChecked] = useState([]);
+  const checkboxes = ['fruits', 'vegetables', 'olive oil'];
 
-  onCheckAll = event => {
-    const { checkboxes } = this.state;
+  const onCheckAll = event => {
     if (event.target.checked) {
-      this.setState({ checked: checkboxes });
+      setChecked(checkboxes);
     } else {
-      this.setState({ checked: [] });
+      setChecked([]);
     }
   };
 
-  onCheck = (event, value) => {
-    const { checked } = this.state;
+  const onCheck = (event, value) => {
     if (event.target.checked) {
-      checked.push(value);
-      this.setState({ checked });
+      setChecked([...checked, value]);
     } else {
-      this.setState({ checked: checked.filter(item => item !== value) });
+      setChecked(checked.filter(item => item !== value));
     }
   };
 
-  render() {
-    const { checked, checkboxes } = this.state;
-
-    return (
-      <Grommet theme={grommet}>
-        <Box align="center" pad="large">
-          <Box direction="row" gap="medium">
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        <Box direction="row" gap="medium">
+          <CheckBox
+            checked={checked.length === 3}
+            indeterminate={checked.length > 0 && checked.length < 3}
+            label="All"
+            onChange={onCheckAll}
+          />
+          {checkboxes.map(item => (
             <CheckBox
-              checked={checked.length === 3}
-              indeterminate={checked.length > 0 && checked.length < 3}
-              label="All"
-              onChange={this.onCheckAll}
+              key={item}
+              checked={checked.indexOf(item) !== -1}
```

Comment:
```suggestion
              checked={checked.includes(item)}
```

Reply:
I tried to push the suggestion myself to save you some time, but I didn't have permission to do it on your branch. 
You can just approve the suggested code instead of committing/pushing it from your local fork. Thanks!